lib/active_triples/rdf_source.rb

```diff
@@ -90,6 +91,8 @@ def type_registry
     # @see RDF::Graph
     # @todo move this logic out to a Builder?
     def initialize(*args, &block)
+      @observers   = Set.new
```

Comment:
I was trying to keep the interface pretty close to `Observable` http://ruby-doc.org/stdlib-2.0.0/libdoc/observer/rdoc/Observable.html, but maybe that's not important?